### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by geertjanw** [apache/netbeans#3430] [General]
Welcome to an awesome new development in the life of Apache NetBeans, which will take us where we've never been before, to a new social media mechanism bringing our discussions and our issues closer to where the code is.

With thanks to Neil C. Smith for making it happen and Apache Airflow for being the inspiration.

Geertjan
Apache NetBeans PMC Chair

**Comment by matthiasblaesing** [comment]
Hi,

Am Samstag, dem 08.01.2022 um 04:03 -0800 schrieb Geertjan Wielenga:
> Welcome to an awesome new development in the life of Apache NetBeans,
> which will take us where we've never been before, to a new social
> media mechanism bringing our discussions and our issues closer to
> where the code is.

maybe I missed something, but in the descriptions, there was talk about
jira, not about moving discussions from mailing list to github. I'm not
conviced, that placing all faith into a single organisation is a good
idea and sourceforge proved, that it will fail at some point.

One quirk that must not be overlooked by this tie in to github: If you
don't agree to their rules, you can't participate. End of story, the
PMC or the ASF can't decide anything in that regard.

Greetings

Matthias

**Reply by neilcsmith-net** [reply]
@matthiasblaesing the on list threads, and the link to Airflow's model / best practice, were very explicit about moving from JIRA to GitHub issues _and discussions_.  With our current code review process, we are already heavily tied in to GitHub, but everything is archived on ASF side too.  We're not replacing the dev@ (or really users@) with this.  I'm going to hide this bit too - how this evolves is a discussion for dev@.
